besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1259,6 +1259,12 @@ static class PermissionsOptionGroup {
       description = "Specifies the number of last blocks to cache  (default: ${DEFAULT-VALUE})")
   private final Integer numberOfblocksToCache = 0;
 
+  @Option(
+      names = {"--rpc-max-trace-range"},
```

Comment:
I think we can renamed it to `--rpc-max-trace-filter-range` as it only applies for this method.

Reply:
Done